draft-normington-jsonpath-latest.xml

```diff
@@ -20,13 +20,13 @@
   <!-- category values: std, bcp, info, exp, and historic
     ipr values: trust200902, noModificationTrust200902, noDerivativesTrust200902,
        or pre5378Trust200902
-    you can add the attributes updates="NNNN" and obsoletes="NNNN" 
+    you can add the attributes updates="NNNN" and obsoletes="NNNN"
```

Comment:
These whitespace changes are distracting. Was this deliberate or just the result of the editor?